ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1195,3 +1195,59 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  hidden: true
+  interval: 12h
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-ci-secret-bootstrap
+  spec:
+    containers:
+    - args:
+      - --bw-password-path=/etc/ci-secret-bootstrap/bw_password
+      - --bw-user=openshift-dev-infra+ci-bitwarden-bot@redhat.com
+      - --config=./core-services/ci-secret-bootstrap/_config.yaml
+      - --dry-run=false
+      - --force=true
+      command:
+      - /usr/bin/ci-secret-bootstrap
+      env:
+      - name: KUBECONFIG
+        value: /etc/build-farm-credentials/sa.config-updater.api.ci.config:/etc/build-farm-credentials/sa.config-updater.app.ci.config:/etc/build-farm-credentials/sa.config-updater.build01.config
+      image: registry.svc.ci.openshift.org/ci/ci-secret-bootstrap:latest
+      imagePullPolicy: Always
+      name: ci-secret-bootstrap
+      resources:
+        requests:
+          cpu: 500m
+      volumeMounts:
+      - mountPath: /etc/ci-secret-bootstrap
+        name: ci-secret-bootstrap
+        readOnly: true
+      - mountPath: /etc/build-farm-credentials
+        name: build-farm-credentials
+        readOnly: true
+      - mountPath: /.config
```

Comment:
Whats the purpose of this?

Reply:
This is for `bw` cli ... nodejs stuff.
You should have it in your $HOME as well:

```
ls ~/.config/configstore
update-notifier-npm-check-updates.json update-notifier-npm.json
```

Otherwise

```
time="2020-05-01T22:13:05Z" level=error msg="bw cmd failed: fs.js:114\n    throw err;\n    ^\n\nError: EACCES: permission denied, mkdir '/.config'\n...
```